pasion_preciouskaye/script.js

```diff
@@ -59,3 +59,60 @@ document.addEventListener('DOMContentLoaded', function () {
     });
   }
 });
+
+function searchCountry() {
+  var countryName = document.getElementById('countryInput').value.trim();
+  if (!countryName) {
+    document.getElementById('countryDetails').innerHTML = '<p>Please enter a country name.</p>';
+    document.getElementById('sameRegionCountries').innerHTML = '';
+    return;
+  }
+
+  fetch('https://restcountries.com/v3.1/name/' + countryName)
+    .then(function(response) {
+      if (!response.ok) {
+        throw new Error('Country not found');
+      }
+      return response.json();
+    })
+    .then(function(countryData) {
+      var country = countryData[0];
+      var details = `
```

Comment:
Always use "let" keyword in declaring variables.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g25d183194fb_0_0